wiki/Modifying-Your-Game-with-Scripts.md

```diff
@@ -37,6 +37,7 @@ To include your Javascript file in your world, first, shut down the World in Fou
 
 # Examples
 Scripts rely on Hooks to automatically execute, see [Hooks](Hooks.md) for system specific Hooks that you can use, or the [Foundry API](https://foundryvtt.com/api/modules/hookEvents.html) for Hooks provided by the Core Foundry Software.  
+The examples below modify different aspects of the dnd5e system's datamodel, to explore this datamodel to see what you can modify, open the Console with F12 and type in `CONFIG.DND5E` then hit Enter.  

```

Comment:
'system configuration' or just 'system config' is probably a better term than 'datamodel', which is something that handles document data.